### PR TITLE
Convert to Python 3 package

### PR DESCRIPTION
**Pull request by taldcroft**
## Description

Convert the fid_drift_mon repo into an installable package that runs in Ska3 and uses modern practices.

## Interface impacts

Stores fid drift data in an SQlite3 file `fid_stats.db3` instead of Sybase.

### Unit tests
No unit tests.

### Functional tests

Planning to compare data values and plots from the new code to those being generated by the Ska2 legacy code.

Test output at https://cxc.cfa.harvard.edu/mta/ASPECT/fid_drift3/

### CODE REVIEW

**Comment by jeanconn** [comment]
Is there a setup.py hanging around in your local git repo for this @taldcroft ?

**Comment by taldcroft** [comment]
Good guess. It's part of this PR now. :smile:

File: fid_drift_mon/copy_index.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import os
```
Comment (jeanconn) [inline fid_drift_mon/copy_index.py:1]: Just do this from plot_drift and get rid of this code.

**Comment by jeanconn** [comment]
Closing this as superseded by #15

**Pull request closed without merging**